templates/database-integrations/data_backed_service/src/data_backed_service/Module.md

```diff
@@ -0,0 +1,71 @@
+# Data Backed Service
+
+Data inside a database can be exposed to the outside world by using a database backed RESTful web service. RESTful API calls enable you to add, view, update, and remove data stored in a database from the outside world. This template demonstrates on building a database-backed RESTful web service with Ballerina.
+
+Please use the guide documentation [Database Interaction](https://github.com/wso2/ballerina-integrator/tree/master/docs/learn/guides/database-integrations/data-backed-service) for a more detailed explanation.
```

Comment:
Better use relative links here. When this is rendered on the doc site it has to point to a doc page, not GitHub.